README.md

```
@@ -1,126 +1,179 @@
+- **Git & GitHub**: 버전 관리 및 협업
+
+- **GitHub Actions**: CI/CD
+
+- **Figma**: UI/UX 디자인 - [Figma 디자인 바로가기](https://www.figma.com/design/XqEZ6dBUOKMbyUnnLNENFP/Mongo-AI-%EC%9E%90%EC%B2%B4-%EB%94%94%EC%9E%90%EC%9D%B8?node-id=0-1&t=5eBEk9YAAfiydj2h-1)
 
 ## 아키텍처 🏗️
 
-MVI + MVVM, Clean Architecture 패턴을 사용합니다.
+MongoAI는 MVI + MVVM, Clean Architecture 패턴을 사용합니다. 이를 통해 확장성, 테스트 용이성, 코드 품질을 보장합니다.
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)